packages/fast-colors/src/colorlib/parseColor.ts

```diff
@@ -158,7 +158,7 @@ export const namedColors: object = {
  * Test if a string looks like a hexadecimal color
  */
 function isColorStringHex(raw: string): boolean {
-    return raw.charAt(0) === "#";
+    return typeof raw === "string" && raw.charAt(0) === "#";

```

Comment:
Assuming we ran into a problem here, do we want to add a test for this?

Reply:
I'm not going to worry about it for now because I have another PR that actually removes this. But yes, running into issues.